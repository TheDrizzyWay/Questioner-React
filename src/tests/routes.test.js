import React from 'react';
import { shallow } from 'enzyme';
import Routes from '../routes';

describe('<Routes />', () => {
    test('should render successfully without crashing', () => {
        const wrapper = shallow(<Routes />);
        expect(wrapper).toMatchSnapshot();
    });
});
