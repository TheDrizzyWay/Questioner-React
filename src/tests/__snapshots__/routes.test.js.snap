// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Routes /> should render successfully without crashing 1`] = `ShallowWrapper {}`;
